crypt4gh/keys.py

```diff
@@ -110,7 +110,7 @@ def generate(seckey, pubkey, callback=None, comment=None, rounds=100):
             LOG.debug('Encoded Private Key: %s', pkey.hex().upper())
         else:
             import sys
-            print(f'WARNING: The private key {seckey} is not encrypted', file=sys.stderr)
+            LOG.warning(f'WARNING: The private key {seckey} is not encrypted', file=sys.stderr)
```

Comment:
Nope again. Keep the `print` statement, it is there on purpose.

Reply:
Cool. Will fix.